samples/layouts/stepper/stepper-overview-example/index.html

```diff
@@ -0,0 +1,190 @@
+  <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" type="text/css" />
+</head>
+
+<body>
+  <div id="root">
+    <div class="container sample">
+      <igc-stepper id="stepper" linear>
+
+        <igc-step invalid>
+          <span slot="title">Card Type</span>
+          <div class="cards-wrapper">
+            <div class="card-wrapper">
+              <igc-badge>Your current choice</igc-badge>
+              <igc-card class="card-first-step">
+                <div class="card-content-wrapper">
+                  <igc-card-media>
+                    <img src="https://www.infragistics.com/angular-demos/assets/images/stepper/card-blue.png" />
+                  </igc-card-media>
+                  <igc-card-content>
+                    <span class="card-currency">400</span>
+                    <span class="card-title">STATEMENT CREDIT OFFER</span>
+                    <p>Business Customized Advanced</p>
+                    <p>Cash Mastercard</p>
+                  </igc-card-content>
+              </igc-card>
+            </div>
+          </div>
+          <div class="card-wrapper">
+            <igc-badge>Your current choice</igc-badge>
+            <igc-card class="card-first-step">
+              <div class="card-content-wrapper">
+                <igc-card-media>
+                  <img src="https://www.infragistics.com/angular-demos/assets/images/stepper/card-red.png" />
+                </igc-card-media>
+                <igc-card-content>
+                  <span class="card-currency">600</span>
+                  <span class="card-title">STATEMENT CREDIT OFFER</span>
+                  <p>Business Travel Advanced</p>
+                  <p>World Mastercard</p>
+                </igc-card-content>
+              </div>
+            </igc-card>
+          </div>
+          <div class="card-wrapper">
+            <igc-badge>Your current choice</igc-badge>
+            <igc-card class="card-first-step">
+              <div class="card-content-wrapper">
+                <igc-card-media>
+                  <img src="https://www.infragistics.com/angular-demos/assets/images/stepper/card-gold.png" />
+                </igc-card-media>
+                <igc-card-content>
+                  <span class="card-currency">500</span>
+                  <span class="card-title">STATEMENT CREDIT OFFER</span>
+                  <p>Business Golden</p>
+                  <p>World Mastercard</p>
+                </igc-card-content>
+              </div>
+            </igc-card>
+          </div>
+    </div>
+    </igc-step>
+    
+    <igc-step invalid>
+      <span slot="title">Business Information</span>
+      <div class="sample-row">
+        <igc-form class="flex-form">
+          <igc-input required label="Name" type="text" name="name"></igc-input>
+          <igc-input required label="Physical address" type="text" name="physicalAddress"></igc-input>
```

Comment:
On each keystoke in this field the focus is moved to the State's igc-select.

Reply:
This is related to the issue [#543](https://github.com/IgniteUI/igniteui-webcomponents/issues/543). It will work properly when the issue is fixed and becomes a part of one of our official releases.